README.md

```diff
@@ -136,6 +138,110 @@ Here is a side-by-side comparison of the input and output video:
     <img src="docs/images/optical-flow.gif" alt="optical-flow-sbs">
 </p>
 
+#### Symbolic audio generation
+
+Create audio sequences by generating symbolic ([MIDI](https://en.wikipedia.org/wiki/MIDI)) audio data and converting the
+generated audio symbols into WAV output using [fluidsynth](https://www.fluidsynth.org/) (_Note:_ fluidsynth must be installed
+in order for the following example to work):  
+
+```python
+from transformers import pipeline
+from pretty_midi import PrettyMIDI
+from perceiver.model.audio import symbolic  # auto-class registration
+
+repo_id = "krasserm/perceiver-ar-sam-giant-midi"
+
+prompt = PrettyMIDI("prompt.mid")
+audio_generator = pipeline("symbolic-audio-generation", model=repo_id)
+
+output = audio_generator(prompt, max_new_tokens=64, num_latents=1, do_sample=True, top_p=0.95, temperature=1.0, render=True)
+
+with open("generated_audio.wav", "wb") as f:
+    f.write(output["generated_audio_wav"])
+```
+
+##### Audio samples
+
+The following (hand-picked) audio samples were generated using various prompts from the validation subset of
+the [GiantMIDI-Piano](https://github.com/bytedance/GiantMIDI-Piano) dataset. The input prompts are
+not included in the audio output.
+
+<table>
+  <tr>
+    <th>Audio sample</th>
+    <th>Top-K</th>
+    <th>Top-p</th>
+    <th>Temperature</th>
+    <th>Prefix length</th>
+    <th>Latents</th>
+  </tr>
+  <tr>
+    <td>
+      <audio controls>
+        <source src="https://martin-krasser.com/perceiver/data/midi/01_nehrlich_continuation.wav" type="audio/wav">
+        Your browser does not support the audio element.
+      </audio>
```

Comment:
My Browser shows the error message. What am I missing?

Reply:
Apparently audio tags are not rendered correctly on GitHub README pages. How about we add a link to the HF model card instead?